### DISCUSSION TITLE
China Confirmed Cases by Province?

### DISCUSSION THREAD

**Discussion by vernhow** [owid/covid-19-data#2705] [Q&A]
Dear Team, would like to know, since I've been using data explorer since the data is aggregated to the country, how can I view the data from the level of province?

**Comment by lucasrodes** [comment]
Hi @vernhow, glad to see that our data is helpful for your work!

We source our data on COVID-19 confirmed cases and deaths from the COVID-19 Data Repository by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University: https://github.com/CSSEGISandData/COVID-19.
This page also lists their sources, country by country. We report this data without alterations, but issues can be raised in their GitHub repository: https://github.com/CSSEGISandData/COVID-19/issues
